equinix/fabric_connection_read_schema.go

```diff
@@ -0,0 +1,708 @@
+			Computed:    true,
+			Description: "Point of access details",
+			Elem: &schema.Resource{
+				Schema: readConnectionSideAccessPointSch(),
+			},
+		},
+	}
+}
+
+func readRedundancySch() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"group": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Redundancy group identifier",
+		},
+		"priority": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Priority type",
+		},
+	}
+}
+
+func readPortRedundancySch() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"priority": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Priority type",
+		},
+	}
+}
+
+func readChangeLogSch() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"created_by": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Created by User Key",
+		},
+		"created_by_full_name": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Created by User Full Name",
+		},
+		"created_by_email": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Created by User Email Address",
+		},
+		"created_date_time": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Created by Date and Time",
+		},
+		"updated_by": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Updated by User Key",
+		},
+		"updated_by_full_name": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Updated by User Full Name",
+		},
+		"updated_by_email": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Updated by User Email Address",
+		},
+		"updated_date_time": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Updated by Date and Time",
+		},
+		"deleted_by": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Deleted by User Key",
+		},
+		"deleted_by_full_name": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Deleted by User Full Name",
+		},
+		"deleted_by_email": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Deleted by User Email Address",
+		},
+		"deleted_date_time": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Deleted by Date and Time",
+		},
+	}
+}
+
+func readOrderSch() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"purchase_order_number": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Purchase order number",
+		},
+		"billing_tier": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Billing tier for connection bandwidth",
+		},
+		"order_id": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Order Identification",
+		},
+		"order_number": {
+			Type:        schema.TypeString,
```

Comment:
Just checking - is this really a string? I see `account_number` is a `schema.TypeInt` so I assume the `_number` API field prefix forecasts the type.

Reply:
It is a string, the order number is generated in backend system, information is propagated to connection api.